### DISCUSSION TITLE
Is is possible to use third party libraries with react native visionos.

### DISCUSSION THREAD

**Discussion by RayanAbid** [callstack/react-native-visionos#120] [Q&A] [answered]
Hi! I was just playing around with this. This is pretty cool so far. I was wondering if we could use third party libs with it. 

I tried using https://github.com/henninghall/react-native-date-picker but it does not work. Is there some difference between the ios and visionos source? 

In short just want to know if it's possible to use third party libs with it. Thanks loving it so far.

**Comment by okwasniewski** [comment, marked as answer]
Hey, yes! It's possible, libraries share most of the iOS code unchanged, but they must add explicit support in their Podspec file. I wrote a short blog post about this process here: https://www.oskarkwasniewski.dev/blog/bringing-react-native-libraries-to-apple-vision-pro

**Reply by RayanAbid** [reply]
Thanks @okwasniewski for sharing this. I will have a look at it. 👌🏻
